package.json

```diff
@@ -17,7 +17,7 @@
     "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
-    "@coinbase/onchainkit": "^0.28.6",
+    "@coinbase/onchainkit": "^0.28.7",
```

Comment:
i think you need to `bun i` as well so lockfile updates

Reply:
Ah thanks I forgot